relevanceai/dataset_api/dataset.py

```diff
@@ -554,6 +575,20 @@ def cat_fields(documents, field_name):
             self.dataset_id, cat_fields, updating_args={"field_name": vector_name}
         )
 
+    def set_cluster_labels(self, vector_field, alias, labels):
+        def add_cluster_labels(documents):
+            docs = self.get_all_documents(self.dataset_id)
+            docs = list(filter(DocUtils.list_doc_fields, docs))
+            set_cluster_field = f"_cluster_.{vector_field}.{alias}"
+            self.set_field_across_documents(
+                set_cluster_field,
+                [f"cluster-{label}" for label in list(labels)],
```

Comment:
I think we have a 
```
    def _label_clusters(self, labels):
        return [self._label_cluster(x) for x in labels]
```

So you should use: 

```
self.label_cluster
```

Reply:
isnt this embedded inside the cluster class?